playground/terraform/infrastructure/private_dns/googleapis_com.tf

```diff
@@ -0,0 +1,52 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+resource "google_dns_managed_zone" "private-zone-private-googleapis" {
+  project     = var.project_id 
+
+  name        = "${var.network_name}googleapis-com"
+  dns_name    = "googleapis.com."
+  description = "Private GoogleApi Zone"
+  
+  visibility = "private"
+
+  private_visibility_config {
+    networks {
+      network_url = var.network_id
+    }
+  }
+}
+
+resource "google_dns_record_set" "a-private-googleapis" {
+  name = "private.googleapis.com."
+  type = "A"
+  ttl  = 300
+
+  managed_zone = google_dns_managed_zone.private-zone-private-googleapis.name
+
+  rrdatas = ["199.36.153.8", "199.36.153.9", "199.36.153.10", "199.36.153.11"]
```

Comment:
Same question, where do these come from?

Reply:
This range is specified in the Private Access doc:
https://cloud.google.com/vpc/docs/configure-private-google-access
Added a comment to clarify